bliss/encoder.py

```diff
@@ -61,13 +61,20 @@ def __init__(
         self.save_hyperparameters()
 
         self.bands = bands
+        self.sdss_bands = ["u", "g", "r", "i", "z"]  # NOTE: SDSS-specific naming conventions
         self.n_bands = len(self.bands)
         self.optimizer_params = optimizer_params
         self.scheduler_params = scheduler_params if scheduler_params else {"milestones": []}
         self.input_transform_params = input_transform_params
 
         self.tile_slen = tile_slen
 
+        # automatically z-score if number of bands is greater than 1.
+        if self.n_bands > 1:
```

Comment:
Seems better to always do what's in the config file (i.e., not do anything automatically)

Reply:
I was thinking that this might be better since we get the weird no prediction behavior when using the non-z-scored multi-band input. It might be an easy mistake to specify multiple bands but forget to specify the z-score transform or some other pre-processing. I can just keep it to what is in the config though if you'd prefer that.

Reply:
How about raising a warning if a user doesn't enable the z-score transform for multiband, instead of automatically turning on z-scoring and ignoring the config file?

Reply:
That's a better idea, sounds good.